capabilities/empowering-teams-to-choose-tools.md

```diff
@@ -1,51 +1,78 @@
+
 
 ## Supporting Practices
 
+The following is a curated list of supporting practices to consider when looking to empower your team to choose tools. While not every practice will be beneficial in every situation, this list is meant to provide teams with fresh, pragmatic, and actionable ideas to support this capability.
+
+
 ### [Establish Golden Paths](/practices/establish-golden-paths.md)
 
-Spotify coined the term [Golden Path](https://engineering.atspotify.com/2020/08/how-we-use-golden-paths-to-solve-fragmentation-in-our-software-ecosystem/) to mean selecting a set of officially supported tools that engineers throughout the org can use. This baseline tooling should be comprehensive enough to address most organizational needs, including programming languages, libraries, testing tools, data storage, and infrastructure. This practice promotes standardization while allowing flexibility for exceptional cases. The benefit is a balanced approach that minimizes the risk of technical debt while allowing teams to focus on delivering value to their stakeholders.
+Spotify coined the term [Golden Path](https://engineering.atspotify.com/2020/08/how-we-use-golden-paths-to-solve-fragmentation-in-our-software-ecosystem/), which means selecting a set of officially supported tools that engineers throughout the organization can use. This baseline tooling should be comprehensive enough to address most organizational needs, including programming languages, libraries, testing tools, data storage, and infrastructure. This practice promotes standardization while allowing flexibility for exceptional cases. The benefit is a balanced approach to tooling that minimizes the risk of technical debt while allowing teams to deliver value to their stakeholders using the best tools available.
 
 ### [Schedule Regular Tooling Audits](/practices/schedule-regular-tooling-audits.md)
 
-Conduct regular audits of the organization's toolset to evaluate its effectiveness and relevance. During these audits, discuss and document the benefits and drawbacks of current tools and explore new technologies. Make sure to invite commentary from all levels of the organization. This practice helps keep the tools aligned with the organization’s evolving goals and fosters a culture of continuous improvement and adaptation.
+Conduct regular audits of the organization's toolset to evaluate its effectiveness and relevance. During these audits, discuss and document the benefits and drawbacks of current tools, and explore new technologies. Make sure to invite commentary from all levels of the organization. This practice helps keep the tools aligned with the organization’s evolving goals and fosters a culture of continuous improvement and adaptation.
```

Comment:
I'd love to understand a general range for what "regular audits" might look like. (e.g., "This might mean monthly, quarterly, or yearly depending on your organization's evolving goals and projects.")

Reply:
It's generic on purpose because the cost/value of the frequency should differ from org to org.